impl/client/src/main/javascript/web/pentaho/visual/models/i18n/model_en.properties

```diff
@@ -283,6 +289,12 @@ LabelsOption.domain.insideBase.f=Inside Base
 LabelsOption.domain.outsideEnd.f=Outside End
 LabelsOption.domain.inside.f=Inside
 LabelsOption.domain.outside.f=Outside
+LabelsOption.domain.insideRight.f=Inside Right
+LabelsOption.domain.insideLeft.f=Inside Left
+LabelsOption.domain.leftTop.f=Left Top
+LabelsOption.domain.leftBottom.f=Left Bottom
+LabelsOption.domain.rightTop.f=Right Top
+LabelsOption.domain.rightBottom.f=Right Bottom
```

Comment:
I believe it is more common to say `Top Left`, `Bottom Right`, ... And by the same reasoning, the domain values themselves could use that convention as well.

Reply:
We are not following E-Chart's conventions. We are defining VizAPI model terms which are independent of any views or third-party libraries, including CCC.
We should follow the same convention used for the other values already part of the `LabelsOption` enum.